### PR TITLE
Use svg demo

### PR DESCRIPTION
**Pull request by marionebl**
* Better quality
* Smaller filesize: before 40kb, after 3kb
* Scriptable: [generate.sh](https://gist.github.com/marionebl/1379477c9fec8b69648fd8ae2a6feb82#file-generate-sh)

### CODE REVIEW

**Comment by marionebl** [comment]
Missing coffeescript errors are unrelated https://travis-ci.org/achojs/acho-skin-cli/builds/323800244#L1198

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <p align="center">
   <br>
-  <img src="https://raw.githubusercontent.com/achohq/acho-skin-cli/master/docs/images/resume.png" alt="demo">
+  <img src="https://cdn.rawgit.com/marionebl/1379477c9fec8b69648fd8ae2a6feb82/raw/14512d55ad9a4e9cfe81e81b524043452d81fedd/example.svg" alt="demo">
```
Comment (Kikobeats) [inline README.md]: can you upload the image to docs/images and link it?

**Review by Kikobeats — CHANGES_REQUESTED** [review]

**Comment by Kikobeats** [comment]
awesome, just be sure to hosted the image with the project 😄

**Comment by marionebl** [comment]
Done, `rawgit` is required to avoid `Content-Type` issues with SVG on Github.

**Review by Kikobeats — APPROVED** [review]

**Comment by Kikobeats** [comment]
Thanks!

**Pull request merged**